libnavigation-base/src/main/java/com/mapbox/navigation/base/options/TripNotificationOptions.kt

```diff
@@ -0,0 +1,97 @@
+package com.mapbox.navigation.base.options
+
+import com.mapbox.navigation.base.TimeFormat
+import com.mapbox.navigation.base.trip.notification.TripNotificationInterceptor
+
+/**
+ * Options for the trip notification.
+ *
+ * Another option that is not listed here, but does change the experience of the trip notification
+ * is the [NavigationOptions.distanceFormatterOptions].
+ *
+ * Warning: These options are ignored when you are using
+ * @MapboxModule(MapboxModuleType.NavigationTripNotification)
+ */
+class TripNotificationOptions private constructor(
+    @TimeFormat.Type val timeFormatType: Int,
+    val interceptor: TripNotificationInterceptor,
```

Comment:
```suggestion
    val interceptor: TripNotificationInterceptor?,
```
Nullability sounds perfectly reasonable here and we could drop the default, empty interceptor.

Reply:
Yeah better than empty